test/behaviour/concept/type/attributetype/AttributeTypeSteps.java

```diff
@@ -123,34 +123,66 @@ public void attribute_type_as_value_type_does_not_have_any_regex(String typeLabe
     }
 
     @Then("attribute\\( ?{type_label} ?) get key owners contain:")
-    public void attribute_type_get_owners_as_key_contains(String typeLabel, List<String> ownerLabels) {
+    public void attribute_type_get_owners_as_key_contain(String typeLabel, List<String> ownerLabels) {
```

Comment:
Drive-by: fix the wording of the existing step methods to align better with the BDD step.